articles/techniques/testing-and-debugging.md

```diff
@@ -180,7 +180,7 @@ Here, the keyword `fail` indicates that the computation should not proceed, rais
 By definition, a failure of this kind cannot be observed from within Q#, as no further Q# code is run after a `fail` statement is reached.
 Thus, if we proceed past a call to `AssertPositive`, we can be assured by that its input was positive.
 
-Building on these ideas, [the prelude](xref:microsoft.quantum.libraries.standard.prelude) offers two especially useful assertions, @"microsoft.quantum.primitive.assert" and @"microsoft.quantum.primitive.assertprob" both modeled as operations onto `()`. These assertions each take a Pauli operator describing a particular measurement of interest, a quantum register on which a measurement is to be performed, and a hypothetical outcome.
+Building on these ideas, <xref:microsoft.quantum.libraries.standard.prelude> offers two especially useful assertions, <xref:microsoft.quantum.intrinsic.assert> and <xref:microsoft.quantum.intrinsic.assertprob> both modeled as operations onto `()`. These assertions each take a Pauli operator describing a particular measurement of interest, a quantum register on which a measurement is to be performed, and a hypothetical outcome.
```

Comment:
What motivated changing `[the prelude](xref:microsoft.quantum.libraries.standard.prelude)` to `<xref:microsoft.quantum.libraries.standard.prelude>`? As currently written, the full title gets brought in by the `<xref:>` notation, rather than just "the prelude" as before.

![image](https://user-images.githubusercontent.com/31516/57874388-4b791580-77c5-11e9-8bc3-7546c1dfb9d1.png)

Reply:
I get it, and changed back to [the prelude]

Reply:
Still shows as full title to me